torch/package/importer.py

```diff
@@ -40,6 +40,7 @@ class Importer(ABC):
         obj2 = getattr(module, obj_name)
         assert obj1 is obj2
     """
+    modules: Dict[str, ModuleType]
```

Comment:
I don't see this being modified anywhere. I'm guessing subclass implementations of `import_module` should add to this dict? Should that be documented somewhere?

It would be nice if there was something like `super().parent_class_method(...)` that took care of this but I don't think that's possible.

Reply:
yeah, the runtime behavior is: if you use the default implementation of whichmodule (which I imagine most importers would), then you need to define `self.modules`. I'll follow up and think of a way to enforce this (sadly ABC can't require that you add an attribute, just a method :/)